src/components/QuestionCardAnswersCount.js

```diff
@@ -31,38 +54,53 @@ function QuestionCardAnswersCount ({ question }) {
     }
   })
 
-  if (answers.length <= 0) return null
-  const hasTrue = trueAnswers > 0
-  const hasFalse = falseAnswers > 0
-  const hasNeither = neitherAnswers > 0
-
   return (
-    <Text>
-      {answers.length} answers
-      (
-      {hasTrue && `${trueAnswers} true`}
-      {hasTrue && (hasFalse || hasNeither) && ', '}
-      {hasFalse && `${falseAnswers} false`}
-      {hasFalse && hasNeither && ', '}
-      {hasNeither && `${neitherAnswers} neither`}
-      )
-    </Text>
+    <RightCardFormat>
+      <Text.Strong>{answers.length} answer</Text.Strong>
+      <StatCountFormat>
+        <p style={{ color: 'green', marginRight: '4px' }}>
+          <FaCheck size='10px' />
+          {` ${trueAnswers} True`}
+        </p>
+        <p style={{ color: 'red', marginRight: '4px' }}>
+          <FaTimes size='10px' /> {`${falseAnswers} False`}
+        </p>
+        <p style={{ color: 'grey' }}>
+          <FaQuestion size='10px' /> {`${neitherAnswers} Neither`}
+        </p>
+      </StatCountFormat>
+      <div>
+        <hr className='solid' background-color='black' />
+        <p>What's you take?</p>
+      </div>
+      <StatCountFormat>
+        <ChoiceButtonFormat onClick={clickMe}>
```

Comment:
Probably want the function name to be descriptive of what's happening, e.g. `handleTrueClick`

Reply:
I think you can safely remove words like `Format` to keep things concise.